{
    "functionName": "test__add_lookup_path_exception",
    "className": null,
    "fileName": "/HDI-Project_&_MLBlocks/tests_&_test_discovery.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "paths = ['a', 'b']\ninvalid_path = str(uuid.uuid4())\nwith pytest.raises(ValueError):\n    discovery._add_lookup_path(invalid_path, paths)\n"
}